js-intermediate.js

```diff
@@ -81,6 +133,10 @@ var myCar = {
     transmission: "manual",
   }
 }
+var doors  = myCar.specs.doors
+console.log(doors)
+//or
+console.log(myCar.specs.doors)
 
```

Comment:
destructuring is:
var { doors } = myCar.specs
What you have works, but the goal here is to practice destructing.